src/App.js

```diff
@@ -19,6 +21,8 @@ const App = () => (
 				<Route path='/logout' component={Logout} />
 				<Route path='/invoke/password-reset' component={InvokePasswordResetEmail} exact />
 				<Route path='/reset/password' component={ResetPassword} exact />
+				<Route path='/profile' component={Profile} exact />
```

Comment:
@collline and @DrKimpatrick, Please use the PrivateRoute instead of Route, It was desigined to handle all routes that require authentication and I think the profile pages need an authenticated user.

Reply:
@reiosantos  we have replaced `Route` with `PrivateRoute`